systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/MultipleListenersST.java

```diff
@@ -71,10 +72,12 @@ void testCombinationOfInternalAndExternalListeners() {
         List<GenericKafkaListener> multipleDifferentListeners = new ArrayList<>();
 
         List<GenericKafkaListener> internalListeners = testCases.get(KafkaListenerType.INTERNAL);
-        List<GenericKafkaListener> nodeportListeners = testCases.get(KafkaListenerType.NODEPORT);
-
         multipleDifferentListeners.addAll(internalListeners);
-        multipleDifferentListeners.addAll(nodeportListeners);
+
+        if (!Environment.isNamespaceRbacScope()) {
+            List<GenericKafkaListener> nodeportListeners = testCases.get(KafkaListenerType.NODEPORT);
+            multipleDifferentListeners.addAll(nodeportListeners);
+        }
```

Comment:
I do not think this makes sense. You are basically turning this into `testMultipleInternal` when running it without the node-ports. Should this just use `Assume` and skip the test in the namespaced mode?